manuscript/conclusion.md

```diff
@@ -107,4 +107,7 @@ My experience level couldn't be as it is now without the help of a lot of people
 ## Other resources
 
 * [OpenSource.guide, a GitHub official project with a lot of guides about Open source, contribute and so on](https://opensource.guide/)
-* [Producing Open Source software, a book by one of the core contributor of SVN](https://producingoss.com/en/)
\ No newline at end of file
+* [Producing Open Source software, a book by one of the core contributor of SVN](https://producingoss.com/en/)
+* [Mozilla Contributor Research 2017](https://mzl.la/whyopensource)
+* [Mozilla Open Strategy 2018](https://mzl.la/openstrategy)
```

Comment:
Be aware, that URL shorteners could stop working in the future. I normally prefer the long URL behind it, but am aware, that URL shorteners could be used for tracking reasons (opening rate).